sympy/combinatorics/polyhedron.py

```diff
@@ -23,14 +26,17 @@ class Polyhedron(Basic):
     References
     ==========
 
-    http://mathworld.wolfram.com/PolyhedralGroup.html
+    .. [1] http://mathworld.wolfram.com/PolyhedralGroup.html
```

Comment:
Please leave an empty line after this too.

Reply:
Done.